### DISCUSSION TITLE
Routes support wildcard

### DISCUSSION THREAD

**Discussion by vongohren** [postalserver/postal#3188] [Feature suggestions]
Did not find any thing in regards to this request.

But it would be great to have a wildcard route that is a catch em all and can be forwarded somewhere. 

Or is this already possible?

**Comment by willpower232** [comment]
Its not immediately obvious but this is already a feature

https://docs.postalserver.io/other/wildcards-and-address-tags

**Reply by vongohren** [reply]
Oh wow great! it was something at the bottom, after a scroll :P Did not find, but thanks for the quick respons
